conf/development.ini

```diff
@@ -28,28 +28,28 @@ log-5xx = true
 
 max-requests = 1000                  # Restart workers after this many requests
 max-worker-lifetime = 3600           # Restart workers after this many seconds
-reload-on-rss = 256                  # Restart workers after this much resident memory (Mb)
+reload-on-rss = 128                  # Restart workers after this much resident memory (Mb)
```

Comment:
Our workers seem to peak at about 70Mb in live. If they got to 128, something is probably wrong